{{cookiecutter.project_slug}}/clients/web/vue3/src/styles/buttons.css

```diff
@@ -1,15 +1,25 @@
 /* Buttons */
 
 .btn {
-  @apply ml-6 inline-flex items-center rounded-md px-3 py-2 text-sm font-semibold shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primaryLight border;
+  @apply flex w-full cursor-pointer items-center justify-center rounded-md border px-3 py-2 text-sm font-semibold shadow-sm;
 }
 
 .btn--primary {
   @apply btn;
-  @apply bg-primary text-white hover:bg-primaryLight border-primary;
+  @apply border-primary text-white hover:bg-primaryLight;
```

Comment:
I removed bg-primary here. I'm not sure what the issue is but the color only seems to be applying to components that are loaded onto a page but not on views. I.e. it worked on the navbar but not on the signup or login pages. I would like to investigate this further but I don't think it should block the work in the meantime